types/office-js-preview/index.d.ts

```diff
@@ -27632,7 +27691,8 @@ declare namespace Excel {
         * Represents the type of this cell value.
         *
         * @remarks
-        * [Api set: ExcelApi 1.16]
+        * [Api set: ExcelApi BETA (PREVIEW ONLY)]
```

Comment:
Wait, how is this moving out of a named requirement set?

Reply:
I don't think this and the following two API set tag changes count as a regression. 

These three have always been within the LocalImageCellValue object, which has always been tagged as BETA: https://learn.microsoft.com/en-us/javascript/api/excel/excel.localimagecellvalue?view=excel-js-preview

I think they were just tagged incorrectly as 1.16 at some point. They were only ever released in the preview office.d.ts (not the prod office.d.ts), and they aren't listed in the 1.16 release article: https://learn.microsoft.com/en-us/javascript/api/requirement-sets/excel/excel-api-1-16-requirement-set?view=common-js-preview.

Reply:
Sounds good. Thanks for verifying.